libs/runtime/ebpf_epoch.h

```diff
@@ -74,10 +74,10 @@ extern "C"
     ebpf_epoch_free(_Frees_ptr_opt_ void* memory);
 
     /**
-     * @Brief Release any memory that is associated with expired epochs.
+     * @brief Wait for the current epoch to end.
      */
     void
```

Comment:
can you mark this with SAL annotation saying it requires being run at PASSIVE level?